### PR TITLE
#6 - 3 questions

### LINKED ISSUE #6: 3 Tree question

### PR DESCRIPTION
**Pull request by niksvat**
closes #6

### CODE REVIEW

File: solutions/count-good-node/Solution.java
Comment (rajrohanyadav) [inline solutions/count-good-node/Solution.java:1]: filename should be `count-good-node.java`

File: solutions/lca/Solution.java
Comment (rajrohanyadav) [inline solutions/lca/Solution.java:1]: please use the complete question name, and update `Solution.java` as well accordingly.

File: solutions/validate-bst/Solution.java
Comment (rajrohanyadav) [inline solutions/validate-bst/Solution.java:1]: the filename should be `validate-bst.java`.

**Review by rajrohanyadav — CHANGES_REQUESTED** [review]
Thanks @niksvat for creating PR.
Please add `<question-name>.txt` file in each of the folders along with URL to the questions.

File: solutions/count-good-node/Solution.java
Comment (niksvat) [reply solutions/count-good-node/Solution.java:1]: Done.

File: solutions/lca/Solution.java
Comment (niksvat) [reply solutions/lca/Solution.java:1]: Done.

File: solutions/validate-bst/Solution.java
Comment (niksvat) [reply solutions/validate-bst/Solution.java:1]: Done.

**Review by rajrohanyadav — APPROVED** [review]

**Pull request merged**